adminutils/adminutils.py

```diff
@@ -189,45 +189,79 @@ async def emoji_add(self, ctx, name: str, url: str, *roles: discord.Role):
         except discord.HTTPException as e:
             await ctx.send(chat.error(_("An error occured on adding an emoji: {}").format(e)))
         else:
+            msg = _("{em} created with the name `{em.name}`".format(em=em))
```

Comment:
```suggestion
            msg = _("{em} created with the name `{em.name}`".format(em=e))
```
`NameError: name 'em' is not defined`